.github/workflows/reusable-doc-test.yaml

```diff
@@ -156,98 +87,9 @@ jobs:
-          restore-keys: |
-            ${{ runner.os }}-buildx-
-
-      - name: Create tags
-        run: |
-          tags="${DOCLINTTEST_IMAGE},${BUILDCACHE_IMAGE}"
-          echo "GITHUB REF: ${GITHUB_REF}"
-          echo "Default Branch: refs/heads/${{ github.event.repository.default_branch }}"
-          echo "runner os: ${{ runner.os }}"
-          echo "github sha: ${{ github.sha }}"
-          echo "build cache image: ${{ env.BUILDCACHE_IMAGE }}"
-          if [[ "${GITHUB_REF}" == "refs/heads/${{ github.event.repository.default_branch }}" ]]; then
-            stableorlatest_image="${IMAGE_NAME}:doclinttest-stable"
-            tags="${tags},${stableorlatest_image}"
-          else
-            stableorlatest_image="${IMAGE_NAME}:doclinttest-latest"
-            tags="${tags},${stableorlatest_image}"
-          fi
-          # PUSH_TAGS is used with the docker buildx action
-          echo "PUSH_TAGS=${tags}"
-          echo "PUSH_TAGS=${tags}" >> $GITHUB_ENV
-          # STABLEORLATEST_IMAGE is only used with the explicit docker build command.
-          # It is the full path to the stable or latest image tag.
-          # The explicit docker build command requires pushing each image separately
-          # to the registry - so we need to track the stable/latest tag in addition
-          # to the DOCLINTTEST_IMAGE and BUILDCACHE_IMAGE tags
-          echo "STABLEORLATEST_IMAGE=${stableorlatest_image}"
-          echo "STABLEORLATEST_IMAGE=${stableorlatest_image}" >> $GITHUB_ENV
-
-      # If using docker buildx was requested via organization variable,
-      # DO NOT run this section.
-      - name: Build and push docker image with explicit docker build
-        if: ${{ vars.USE_DOCKER_BUILDX_ACTION == 'false' }}
-        # > makes newlines into spaces, so single command on multiple lines.
-        run: >
-          which docker;
-          docker build
-          --file Dockerfile
-          --tag ${DOCLINTTEST_IMAGE}
-          --tag ${BUILDCACHE_IMAGE}
-          --tag ${STABLEORLATEST_IMAGE}
-          --target doclinttest
-          .;
-          docker push ${BUILDCACHE_IMAGE};
-          docker push ${DOCLINTTEST_IMAGE};
-          docker push ${STABLEORLATEST_IMAGE};
-      - name: Build and push docker image with docker buildx action
-        if: ${{ vars.USE_DOCKER_BUILDX_ACTION == 'true' }}
-        uses: docker/build-push-action@v6
-        with:
-          context: .
-          target: doclinttest
-          push: true
-          tags: ${{ env.PUSH_TAGS }}
-          file: "Dockerfile"
-          cache-from: type=registry,ref=${{ env.BUILDCACHE_IMAGE }}
-          cache-to: type=registry,ref=${{ vars.DOCKER_REGISTRY }}/${{ env.BUILDCACHE_IMAGE }},mode=max
+    name: Build docker image
+    # You do not appear to be able to use variables in this "uses" field.
+    uses: NRLMMD-GEOIPS/geoips_ci/.github/workflows/reusable-build-docker-image.yaml@build_stable_image_on_tag
```

Comment:
This is all it needed to still handle the docker images properly!!

Reply:
And it still works, see data_fusion repository (successful doc-test.yaml):

https://github.com/NRLMMD-GEOIPS/data_fusion/actions/runs/12698134825?pr=39

![image](https://github.com/user-attachments/assets/b2995b90-21be-4a96-a5a0-c5f6ab400fcc)